dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -93,10 +93,7 @@ cdi12.element.type.superceded.CWOWB1015W=CWOWB1015W: The cdi12 configuration ele
 cdi12.element.type.superceded.CWOWB1015W.explanation=The cdi configuration element replaces the cdi12 configuration element and contains all the same options, plus new options.
 cdi12.element.type.superceded.CWOWB1015W.useraction=Change your server.xml configuration to use the cdi configuration element instead of the cdi12 configuration element.
 
-#CWOWB1016W is not used. The emptyBeansXMLExplicitBeanArchive config option is silently ignored if not running CDI 4.0+
-#cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W=CWOWB1016W: The emptyBeansXMLExplicitBeanArchive attribute of the cdi configuration element is supported only on CDI 4.0 or newer. This attribute is ignored.
-#cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W.explanation=The emptyBeansXMLExplicitBeanArchive attribute has no effect when it is used with a CDI version lower than 4.0.
-#cdiEmptyBeansXMLExplicitBeanArchive.ignored.CWOWB1016W.useraction=Update your server.xml configuration to either remove the attribute or use CDI version 4.0 or newer.
+#CWOWB1016W is not used - https://github.com/OpenLiberty/open-liberty/pull/21107
```

Comment:
Why do you have to mention this? I would just remove it. If you want, add this to the issue itself not in this file.